### ISSUE TITLE
No trigger

### ISSUE THREAD

**Issue by ametthey** [Stanko/animated-scroll-to#43]
Hello, I'm trying to make your plugin work for me without any chance for now..

Here is my vanilla JS code

![Capture d’écran 2020-04-20 à 17 12 55](https://user-images.githubusercontent.com/7902934/79767833-33d9d380-832a-11ea-8d8f-5ced0bfb2e6c.jpg)

And render HTML in google chrome console,

![Capture d’écran 2020-04-20 à 17 13 39](https://user-images.githubusercontent.com/7902934/79768103-9632d400-832a-11ea-9d5f-4fb48fa02080.jpg)

In plain english, what I am trying to achieve is :

When I click on the button with the class **js-trigger**, the container **#section--vertical** will scroll to the element **#slide__project--first**

Perhaps I am using your function **animateScrollTo** wrong ...

Can you help me out ?

**Comment by Stanko** [comment]
Hello @ametthey, from the screenshots it looks like you just swapped `target` and `container`.

Function has multiple signatures (which can be confusing), but you need to use this one:
```js
function animateScrollTo(scrollToElement, options);
````

Here is an example:
https://codesandbox.io/s/relaxed-swartz-b5me8?file=/src/index.js

**Comment by ametthey** [comment]
Holy sh** thanks a lot for that ! I agree that the doc is a bit confusing but now it's all clear ! Thanks for your time !

**Comment by Stanko** [comment]
No worries 🙈 it happens, I'm looking into adding a warning message it `target` is not a child of the `container`.

**Comment by Stanko** [comment]
I added a check in the new version 2.0.6

https://github.com/Stanko/animated-scroll-to/blob/master/src/animated-scroll-to.ts#L203-L212

Cheers!
